__tests__/app.test.js

```diff
@@ -49,3 +49,38 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("status:200, responds with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(String),
+            })
+          );
+          expect(articles).toBeSortedBy("created_at", { descending: true });
```

Comment:
Good use of toBeSortedBy, I would consider either changing the test description to mention sorting OR if following best practice; create another test that tests the sorting functionality, when using TDD each test should test one piece of functionality.

Reply:
Not a huge issue though, you can still merge 👍

Reply:
Thanks Rob, I'll put this into place now